container/go/cmd/puller/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
Also add license header in the BUILD file as well this line licenses(["notice"])  # Apache 2.0

similar to https://github.com/bazelbuild/rules_docker/blob/master/BUILD